static/docs/command-reference/run.md

```diff
@@ -52,25 +52,25 @@ captures data and <abbr>caches</abbr> relevant <abbr>data artifacts</abbr> along
 the way. See [this example](/doc/get-started/example-pipeline) to learn more and
 try creating a pipeline.
 
-### Well-behaved commands
-
-DVC is simple to use, you only need to wrap your commands with `dvc run`, and
-define your dependencies and outputs.
-
-However, to prevent unexpected behaviors, ideally, your commands should follow
-some principles:
-
-- Read exclusively from specified dependencies.
-- Write exclusively to specified outputs.
-- Completely rewrite the outputs (i.e. do not append or edit).
-- Stop reading and writing when the command exits.
-
-To guarantee reproducibilty, your command should be
-[deterministic](https://en.wikipedia.org/wiki/Deterministic_algorithm) (i.e. it
-must produce the same results given the same inputs/dependencies).
-
-Have in mind what brings entropy to your command (e.g. random generators, time,
-hardware, etc.) and try to minize it (e.g. fix seeds).
+### Reproducibility notes
+
+We don't want to tell you how to write your code! However, please be aware that
+in order to prevent unexpected results when DVC executes or reproduces your
+commands, they should ideally follow these rules:
+
+- Read/write exclusively from/to the specified <abbr>dependencies</abbr> and
+  <abbr>outputs</abbr>.
+- Completely rewrite outputs (i.e. do not append or edit).<br/> Note that DVC
+  removes cached outputs before running the stages that produce them (including
+  at `dvc repro`).
+- Stop reading and writing files when the `command` exits.
```

Comment:
minor: probably remove ticks `command` -> command since we don't emphasize it before?

Reply:
`command` is a `dvc run` argument and it's surrounded with back ticks several times above in the description. But "command" is also just a word that I use without back ticks in some other sentences.

Reply:
But reviewing this I noticed some inconsistencies and errors in sentences around this term. Updated in 884bdef4.